cli/src/pcluster/templates/cdk_builder.py

```diff
@@ -34,7 +36,7 @@ def build_cluster_template(cluster_config: BaseClusterConfig, bucket: S3Bucket,
         with tempfile.TemporaryDirectory() as tempdir:
             output_file = str(stack_name)
             app = App(outdir=str(tempdir))
-            ClusterCdkStack(app, output_file, stack_name, cluster_config, bucket)
+            ClusterCdkStack(app, output_file, stack_name, cluster_config, bucket, log_group_name=log_group_name)
```

Comment:
Can we avoid the keyword, to be consistent with other parameters, or add the keyword for all of them?